app/pb_kits/playbook/pb_typeahead/docs/_typeahead_with_pills.jsx

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+
+import React from 'react'
+import { Typeahead } from '../..'
+
+const options = [
+  { label: 'Windows', value: '#FFA500' },
+  { label: 'Siding', value: '#FF0000' },
+  { label: 'Doors', value: '#00FF00' },
+  { label: 'Roofs', value: '#0000FF' },
```

Comment:
I recall that react-select had an option to use different keys for the options. Is it still possible? 

In Nitro, we will use a lot of `id` and `name` instead of `label` and `value`.

Reply:
Yeh I do believe this is customizable. I will have a look.

Reply:
@gmfvpereira [this is how you would change `value`](https://codesandbox.io/s/zegbp?module=/example) and [this is how you would change `label`](https://codesandbox.io/s/yi20c?module=/example) to use different fields. You can send these as props to `<Typeahead>`.